src/test/java/com/kenai/jffi/VersionTest.java

```diff
@@ -0,0 +1,44 @@
+package com.kenai.jffi;
+
+import org.junit.Test;
+
+import com.kenai.jffi.InvokerTest.HeapInvoker;
+import com.kenai.jffi.InvokerTest.NativeInvoker;
+import com.kenai.jffi.UnitHelper.Address;
+
+import static org.junit.Assert.assertEquals;
+
+public class VersionTest {
+
+    public VersionTest() {
+    }
+
+    @Test public void old_answer() {
+        Invoker invoker = new NativeInvoker();
+
+        Address sym = UnitHelper.findSymbolWithVersion("answer", "VERS_1.0");
+        Function function = new Function(sym.address, Type.SINT);
+        CallContext ctx = new CallContext(Type.SINT);
+
+        long res = invoker.invokeN0(ctx, function.getFunctionAddress());
+
+        assertEquals(41, res);
+    }
+
+    @Test public void new_answer() {
+        Invoker invoker = new HeapInvoker();
```

Comment:
Same remark as above.  I used the NativeInvoker above so I chose the HeapInvoker here, but does it makes sense?

Reply:
Whatever works for the test is fine.